spec/features/project_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe Project do
+  before do
+    # challenge
+    @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    @furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+    #projects
+    @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+
+    @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+    #contestants
+    @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+    @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+    #contestant projects
+    ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+    ContestantProject.create(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+    ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+    ContestantProject.create(contestant_id: @erin.id, project_id: @boardfit.id)
+  end
+
+
+  # User Story 1 of 3
+  #(e.g.    Litfit
+  #     Material: Lamp Shade
+  #   Challenge Theme: Apartment Furnishings)
+
+  describe "As a visitor," do
+    describe "When I visit a projects show page (/projects/:id)," do
+      describe "I see that projects name and material" do
+        it "And I also see the theme of the challenge that this project belongs to" do
+
+          visit ("/projects/#{@news_chic.id}")
+
+
+          expect(page).to have_content("News Chic")
+          expect(page).to have_content("Newspaper")
+          expect(page).to have_content("Recycled Material")
+        end
+      end
+    end
+  end
+
+  # User Story 3 of 3
+  # (e.g.    Litfit
+  #   Material: Lamp Shade
+  # Challenge Theme: Apartment Furnishings
+  # Number of Contestants: 3 )
+  describe "As a visitor" do
+    describe "When I visit a project's show page" do
+      it "I see a count of the number of contestants on this project" do
+
+        visit ("/projects/#{@news_chic.id}")
+
+        expect(page).to have_content("News Chic")
+        expect(page).to have_content("Newspaper")
+        expect(page).to have_content("Recycled Material")
+        expect(page).to have_content(2)
```

Comment:
Same feedback here -- how can you make this assertion more specific so that you're sure this 2 is referencing the number of contestants and isn't just a random floating 2?